style.css

```diff
@@ -167,4 +167,23 @@ body {
   font-size: 10px;
   text-align: center;
   color: rgb(69, 69, 69);
+}
+
+.logo {
+    display: flex;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**CSS file**

```
.flex{
  display:flex;
}
```
**Html file**

```
<div class="flex">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
We change the classes and as you requested and fixing the padding too, also we change the main section to <main> tag.